fitters/include/DAF.h

```diff
@@ -60,7 +60,13 @@ class DAF : public AbsKalmanFitter {
    *
    * @param useRefKalman If false, use KalmanFitter as fitter.
    */
-  DAF(bool useRefKalman = true, double deltaPval = 1e-3, double deltaWeight = 1e-3);
+  DAF(bool useRefKalman = true, double deltaPval = 1e-3, double deltaWeight = 1e-3, double probCut = 1e-3, std::tuple<double, double, int> annealingScheme = std::tuple<double, double, int>(100., 0.1, 5), int minIter = 1, int maxIter = 10, int minIterForPval = 1);
+  /**
+   * @brief Create DAF. Per default, use KalmanFitterRefTrack as fitter.
+   *
+   * @param useRefKalman If false, use KalmanFitter as fitter.
+   */
+  DAF(bool useRefKalman, double deltaPval, double deltaWeight);
```

Comment:
Keep the old DAF constructor as it is in the original version with the default values